src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -0,0 +1,1024 @@
+#else
+    if(!conduit::blueprint::verify("mesh", _mdMesh, info))
+#endif
+    {
+      SLIC_INFO("Invalid blueprint for mesh: \n" << info.to_yaml());
+      slic::flushStreams();
+      return false;
+    }
+    // info.print();
+    return true;
+  }
+
+  void print_mesh_info() const { _mdMesh.print(); }
+
+private:
+  int _ndims {-1};
+  conduit::Node _mdMesh;
+  axom::IndexType _domCount;
+  const std::string _coordsetPath;
+  const std::string _topologyPath;
+
+  /*!
+    @brief Read a blueprint mesh.
+  */
+  void read_blueprint_mesh(const std::string& meshFilename)
+  {
+    SLIC_ASSERT(!meshFilename.empty());
+
+    _mdMesh.reset();
+#ifdef AXOM_USE_MPI
+    conduit::relay::mpi::io::blueprint::load_mesh(meshFilename,
+                                                  _mdMesh,
+                                                  MPI_COMM_WORLD);
+#else
+    conduit::relay::io::blueprint::load_mesh(meshFilename, _mdMesh);
+#endif
+    assert(conduit::blueprint::mesh::is_multi_domain(_mdMesh));
+    _domCount = conduit::blueprint::mesh::number_of_domains(_mdMesh);
+
+    if(_domCount > 0)
+    {
+      const conduit::Node coordsetNode = _mdMesh[0].fetch_existing(_coordsetPath);
+      _ndims = conduit::blueprint::mesh::coordset::dims(coordsetNode);
+    }
+#ifdef AXOM_USE_MPI
+    MPI_Allreduce(MPI_IN_PLACE, &_ndims, 1, MPI_INT, MPI_MAX, MPI_COMM_WORLD);
+#endif
+    SLIC_ASSERT(_ndims > 0);
+
+    bool valid = isValid();
+    SLIC_ASSERT(valid);
+  }
+};  // BlueprintStructuredMesh
+
+/// Output some timing stats
+void print_timing_stats(axom::utilities::Timer& t, const std::string& description)
+{
+  auto getDoubleMinMax =
+    [](double inVal, double& minVal, double& maxVal, double& sumVal) {
+#ifdef AXOM_USE_MPI
+      MPI_Allreduce(&inVal, &minVal, 1, MPI_DOUBLE, MPI_MIN, MPI_COMM_WORLD);
+      MPI_Allreduce(&inVal, &maxVal, 1, MPI_DOUBLE, MPI_MAX, MPI_COMM_WORLD);
+      MPI_Allreduce(&inVal, &sumVal, 1, MPI_DOUBLE, MPI_SUM, MPI_COMM_WORLD);
+#else
+      minVal = inVal;
+      maxVal = inVal;
+      sumVal = inVal;
+#endif
+    };
+
+  {
+    double minCompute, maxCompute, sumCompute;
+    getDoubleMinMax(t.elapsedTimeInSec(), minCompute, maxCompute, sumCompute);
+
+    SLIC_INFO(axom::fmt::format("'{}' took {{avg:{}, min:{}, max:{}}} seconds",
+                                description,
+                                sumCompute / numRanks,
+                                minCompute,
+                                maxCompute));
+  }
+}
+
+/**
+ * Change cp_domain data from a local index to a global domain index
+ * by adding rank offsets.
+ * This is an optional step to transform domain ids verification.
+ */
+void add_rank_offset_to_surface_mesh_domain_ids(
+  int localDomainCount,
+  axom::mint::UnstructuredMesh<axom::mint::SINGLE_SHAPE>& surfaceMesh)
+{
+#ifdef AXOM_USE_MPI
+  // perform scan on ranks to compute totalNumPoints, thetaStart and thetaEnd
```

Comment:
Possible typo: 
I don't see `totalNumPoints`, `thetaStart` or `thetaEnd`. 
Please update the comment.

Reply:
Cut/paste error.  Thanks for catching.